src/middlewares/request.middleware.js

```diff
@@ -295,10 +295,28 @@ function checkChannelID (req, res, next) {
   next()
 }
 
+function addChannelFilters (req, res, next) {
+  // Assuming that the request will not send the filter of channels
+  if (req && req.body && req.body.request && req.body.request.filters && req.body.request.filters.channel) {
+    next()
+  } else {
+    filterService.getChannelSearchString(function (err, channels) {
```

Comment:
@rajeevsathish : Do we still need filterService? why cant we directly use "configUtil.getConfig('CHANNEL_FILTER_QUERY_STRING')"?

Reply:
@vrayulu in this release we had only channel filter but in 1.8 we are planning to put more filter so have created filterService